src/content/docs/docs/support/help-us.mdx

```diff
@@ -1,16 +1,11 @@
 ---
 title: Help Us
 ---
-This page exists for us to track changes requested by users who don't necessarily want to create a PR to update the documentation but have suggestions.
 
-## Not Documented
+The Xahau documentation is open source and open for contributions [on GitHub](https://github.com/Xahau/xahau-web).
 
-* accounts on xahau start at a sequence that matches the timestamp of the previous ledger
-* Pseudo tx docs missing: UNLReport, EmitFailure, Etc
-* URIToken Size Limit
-* AccountDelete is still listed as a transaction but is disabled on Xahau
+A lot of gaps and uncovered details are expected in the documentation, so we invite **everyone** to collaborate on bringing it up to standard.
 
-## Suggestions
+Do you see anything that is missing, please create an issue on the [GitHub Project board](https://github.com/orgs/Xahau/projects/2/views/1) or just go ahead and create a pull request to the main branch, with suggested changes or updated documentation.
```

Comment:
It may be better tomake issues rather than adding them to the project board, and then have authorized persons organize the contents and add them to the project board.

And in this case, it is common to use "issues"